Floofbot/Modules/Utilities.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
 using System.Threading.Tasks;
 using Discord;
+using Discord.Addons.Interactive;
 using Discord.Commands;
 using Discord.WebSocket;
 
 namespace Floofbot
 {
-    public class Utilities : ModuleBase<SocketCommandContext>
+    public class Utilities : InteractiveBase
     {
         private static readonly Discord.Color EMBED_COLOR = Color.Magenta;
```

Comment:
We'll need to figure out these colours and where to place them in a future PR considering that the serverlist command has a dark green (admin) embed colour right now.

Reply:
Lol didn't even notice this I can swap it to the random colour?